tests/cypress/integration/wp-module-support/siteGen.cy.js

```diff
@@ -34,3 +34,36 @@ export  const OptionsDetails = (className,textValue,optionsValue) => {
             .click();
     };
 };
+
+export const ProgressBarCheck = ( WidthPercent ) => {
+    cy.get('.nfd-onboarding-header__progress-bar').should('be.visible');
+    cy.get('.nfd-onboarding-header__progress-bar__progress')
+      .invoke('attr', 'style')
+      .then((styleAttribute) => {
+        const value = styleAttribute.match(/width:\s*([\d.]+%)/)[1];
+        expect(value).equal(WidthPercent);
+      });
+};
+
+export const BackButtonCheck = (currURL) => {
+    cy.get('.nfd-onboarding-button--dark')
+        .should('be.visible')
+        .click();
+    cy.url().should('not.contain', currURL);
+    cy.go('back');
+};
```

Comment:
Click Back meaning you are in site-detail step and cy.go('back') will take us to site-logo again right? Just checking.

Reply:
Yess.